docs/ResourceInterface.md

```diff
@@ -131,14 +104,30 @@ public static function relationships(): RelationshipsCollection
 }
 ```
 
-### memoizeRelationships
-In previous example we generate new objects on each `relationships` method call.
-We can use memoize pattern so that we will build the relationships list once and save them in memory and return on request.
+The `ToManyRelationship::create()` method accepts 4 parameters:
+
+- `name` (required):
+
+Name of the relationship that will be used in the endpoint:
+
+i.e: `GET /user/1/{name}` or `GET /user/1/relationships/{name}`
+
+i.e: `GET /user/1/roles` or `GET /user/1/relationships/roles`
 
-For that `ResourceInterface` must use [Sowl\JsonApi\Relationships\MemoizeRelationshipsTrait](../src/Relationships/MemoizeRelationshipsTrait.php).
+- `class` (required):
 
-The Trait provides static method `memoizeRelationships(callback $cb): RelationshipsCollection`.
-Method must receive callback that will return relationships' collection.
+The entity (resource) class being the relation.
+
+- `mappedBy` (required):
+
+Name of the association-field on the owning side of the relation.
+
+- `property` (optional):
```

Comment:
It must be improved

It's can be something like:

"Override the relationship property name that will be the {name} by default."
for example if you have `protected Country $userRoles` but you want to have simple "country" relationship name, you need to provide "userRoles" as a property param value.